backend/src/api/SiteController.ts

```diff
@@ -51,7 +51,7 @@ export default class SiteController {
             bookmarks: Joi.boolean().default(false)
         });
         const siteCreateSchema = Joi.object<SiteCreateRequest>({
-            site: Joi.string().regex(/^[a-z\d-]{3,10}$/i).required(),
+            site: joiSite.required(),
```

Comment:
Can reuse `joiSite` validator for `site` field in the other API methods (above).